### PR TITLE
remove obsolete config pessimistic txn ttl

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### CODE REVIEW

**Comment by mahjonp** [comment]
LGTM
PTAL @liubo0127 @tiancaiamao

File: roles/tidb/vars/default.yml [hunk 1]
```diff
@@ -179,7 +179,3 @@ pessimistic_txn:
 
   # max retry count for a statement in a pessimistic transaction.
   max-retry-count: 256
-
-  # default TTL in milliseconds for pessimistic lock.
-  # The value must between "15s" and "60s".
-  ttl: "30s"
```
Comment (tiancaiamao) [inline roles/tidb/vars/default.yml]: LGTM, but we have to make sure rolling updates with the old configures also work.

Comment (mahjonp) [reply roles/tidb/vars/default.yml]: link https://github.com/pingcap/tidb/pull/13142

**Comment by liubo0127** [comment]
LGTM. The user basically only modifies the parameters in the `conf/tidb.yml`, and `ttl` generally does not change. The parameters in the `roles/tidb/vars/default.yml` are only used as default values. After removing `ttl`, there should be no problems.

**Pull request merged**